Tests/ApodiniTests/EnvironmentTests.swift

```diff
@@ -0,0 +1,54 @@
+//
```

Comment:
Thanks for the unit test! There are a few errors in the test coverage report so I would suggest to add additional tests for some edge cases to improve the test coverage.